### ISSUE TITLE
bug: no/empty values.yaml causes panic in 'helm inspect'

### ISSUE THREAD

**Issue by mumoshu** [kubernetes/helm#1024]
Hi, thanks for developing and sharing a great software!

With the `helm` built from current master, when I run `helm inspect` against a package which (1) [has no `values.yaml`](https://github.com/mumoshu/charts/tree/cd5bc11ad06fd4ba18096552f367621592e368dd/mysql) or (2) [has a empty `values.yaml`](https://github.com/mumoshu/charts/blob/288a9ac0513e11e075ada1d5dec0a01cdb8e611f/mysql/values.yaml), it ends up with a panic.
Is it an expected behavior?

```
$ helm inspect mumoshu/mysql-0.2.1.tgz
Fetched mumoshu/mysql-0.2.1.tgz to /Users/me/PublicRepositories/charts/mysql-0.2.1.tgz
description: Chart running MySQL.
home: https://www.mysql.com/
maintainers:
- email: mfisher@deis.com
  name: Matt Fisher
name: mysql
version: 0.2.1

---
panic: runtime error: invalid memory address or nil pointer dereference
[signal 0xb code=0x1 addr=0x0 pc=0x9759]

goroutine 1 [running]:
panic(0x16c1f20, 0xc820010100)
	/usr/local/Cellar/go/1.6.2/libexec/src/runtime/panic.go:481 +0x3e6
main.(*inspectCmd).run(0xc82029bd40, 0x0, 0x0)
	/Users/me/Gopath/src/k8s.io/helm/cmd/helm/inspect.go:138 +0x599
main.newInspectCmd.func1(0xc820482d80, 0xc82022b530, 0x1, 0x1, 0x0, 0x0)
	/Users/me/Gopath/src/k8s.io/helm/cmd/helm/inspect.go:79 +0x170
k8s.io/helm/vendor/github.com/spf13/cobra.(*Command).execute(0xc820482d80, 0xc82022b4f0, 0x1, 0x1, 0x0, 0x0)
	/Users/me/Gopath/src/k8s.io/helm/vendor/github.com/spf13/cobra/command.go:571 +0x668
k8s.io/helm/vendor/github.com/spf13/cobra.(*Command).ExecuteC(0xc8204fb440, 0xc820482d80, 0x0, 0x0)
	/Users/me/Gopath/src/k8s.io/helm/vendor/github.com/spf13/cobra/command.go:661 +0x55c
k8s.io/helm/vendor/github.com/spf13/cobra.(*Command).Execute(0xc8204fb440, 0x0, 0x0)
	/Users/me/Gopath/src/k8s.io/helm/vendor/github.com/spf13/cobra/command.go:620 +0x2d
main.main()
	/Users/me/Gopath/src/k8s.io/helm/cmd/helm/helm.go:104 +0x23
```

You can reproduce it with my charts repository(containing charts for helm v2.0.0-alpha.2):

```
$ helm repo add mumoshu https://mumoshu.github.io/charts
$ helm inspect mumosh/mysql-0.2.0.tgz # missing values.yaml in the package. fails.
$ helm inspect mumoshu/mysql-0.2.1.tgz # contains an empty values.yaml file. fails.
$ helm inspect mumoshu/mysql-0.2.1.tgz # contains a dummy, non-empty values.yaml file. succeeds.
```

**Comment by technosophos** [comment]
Hmmm... it doesn't seem like we should disallow an empty values file. And we certainly shouldn't panic. I'm going to upgrade this to a bug.

**Comment by technosophos** [comment]
BTW -- we love the way you used a github page for hosting the chart repository. That made it really easy for us to test. Thank you!

**Comment by technosophos** [comment]
I believe this is closed by #1025
